### ISSUE TITLE
Examples incorrect in user manual

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#4445]
SANDLAS were following the example in 
https://github.com/ISISComputingGroup/ibex_user_manual/wiki/genie_python-and-IBEX-(Common-commands) but it is using incorrect syntax. It says:
```
name = block_info("name")
```
it should be 
```
name = block_info["name"]
```

Fix and check other examples

**Comment by John-Holt-Tessella** [comment]
Correct example, other error should be found. In general if you see an error correct it.

**Comment by DominicOram** [comment]
Is this now done?

**Comment by John-Holt-Tessella** [comment]
No, I haven't checked other examples, I just fixed that one.

**Comment by Alexandru-Dascalu** [comment]
I used the command grep -r "c.get" /ibex_user_manual.wiki/ to search the user wiki. I searched for cget as that was the command that returned the dictionary in the original mistake.

Searching for occurrences of anything between parantheses, square brackets or for g. would have taken way too much time, as they appear in a lot of places in the wiki.

The only other place where I could find the mistake was the Test RST page, which I have fixed.
